tests/ceph_ansible/filestore_to_bluestore.py

```diff
@@ -16,6 +16,7 @@ def run(**kw):
     for cnode in ceph_nodes:
         if cnode.role == "installer":
             installer_node = cnode
```

Comment:
We could do this way... 
```
installer_node = kwargs["ceph_cluster"].get_nodes(role="installer")[0]

for cnode in ceph_nodes:
  ...
```

Reply:
@Ameenasuhani , I do suggest same fix as @psathyan pointed out. Since there is possibility of `installer_node` not being set and its good to fail there itself.